### DISCUSSION TITLE
FFmpeg returned an error and how to fix it

### DISCUSSION THREAD

**Discussion by MuBai-He** [akiver/cs-demo-manager#693] [Q&A] [answered]
![image](https://github.com/akiver/cs-demo-manager/assets/51275418/4d3b172e-e224-4817-9213-2af03b2b23b3)

**Comment by MuBai-He** [comment]
![image](https://github.com/akiver/cs-demo-manager/assets/51275418/90c0e545-f94b-4424-8551-fd0c989af115)

**Reply by akiver** [reply]
Could you please upload the demo and give sequence details (start tick, end tick...)?
In your screenshot, there is one sequence of only 2 seconds, which is weird?

**Comment by MuBai-He** [comment]
![image](https://github.com/akiver/cs-demo-manager/assets/51275418/203e3d4b-a1e9-40c9-b3de-dedd53a2b64c)
I'm sure there is no problem with the demo I provided.Then I generated other fragments.
![image](https://github.com/akiver/cs-demo-manager/assets/51275418/0d7a1c84-e667-4701-91ce-8705f2a37f08)
It shows another error like:
![image](https://github.com/akiver/cs-demo-manager/assets/51275418/1529dd3d-4fe0-4434-97ad-0033f7c00ccb)

**Reply by akiver** [reply]
> I'm sure there is no problem with the demo I provided.

You didn't provide any demo, please send a link to download it.

**Comment by MuBai-He** [comment]
Here:https://drive.google.com/file/d/1B8sdUSQWg_SwQUfHEpXRJ20dJC211CsQ/view?usp=sharing

**Reply by akiver** [reply, marked as answer]
Thanks I think I found the problem, it's tracked  in https://github.com/akiver/cs-demo-manager/issues/701

**Reply by MuBai-He** [reply]
Thanks for your help!Have a nice day!
